docs/samples/Microsoft.ML.Samples/Dynamic/ProjectionTransforms.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// the alignment of the usings with the methods is intentional so they can display on the same level in the docs site.
```

Comment:
remove those. Not needed for samples. another PR is removing them for the existing files.